docs/cookbook/displaying_children_of_a_content_item.md

```diff
@@ -1,21 +1,81 @@
 # Displaying children of a Content item
 
-## Description
-
-One of the basic design tasks you may need to complete when creating your website is configuring one page to display all of its children. Examples are having a blog display blog posts, of a folder that shows all articles it contains.
-
-## Solution
+One of the basic design tasks you may need to complete when creating your website is configuring one page to display all of its children. Examples are having a blog display blog posts, or a folder that shows all articles it contains.
 
 There are two ways to make a Content item display its children:
 
-1.  [Using a Custom Controller](#using-a-custom-controller)
-2.  [Using the Query Controller](#using-the-query-controller)
+1. [Using the Query Controller](#using-the-query-controller)
+1. [Using a Custom Controller](#using-a-custom-controller)
 
 This recipe will show how to use both those methods to display all children of a Content item with the Content Type Folder.
 
-### Using a Custom Controller
+## Using the Query Controller
+
+The Query Controller is a predefined custom content view Controller that runs a Repository Query.
+
+If you need to create a simple query it's easier to use the Query Controller than to build a completely custom one, as you will not have to write custom PHP code. Like with a [Custom Controller](#using-a-custom-controller), however, you will be able to use properties of the viewed Content or Location as parameters.
+
+The main file in this case is an `AppBundle/QueryType/LocationChildrenQueryType.php` file which generates a Query that retrieves the children of the current Location.
+
+``` php
+namespace AppBundle\QueryType;
+
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\ParentLocationId;
+use eZ\Publish\Core\QueryType\QueryType;
+
+class LocationChildrenQueryType implements QueryType
+{
+    public function getQuery(array $parameters = [])
+    {
+        return new LocationQuery([
+            'filter' => new ParentLocationId($parameters['parentLocationId']),
+        ]);
+    }
+
+    public function getSupportedParameters()
+    {
+        return ['parentLocationId'];
+    }
+
+    public static function getName()
+    {
+        return 'LocationChildren';
+    }
+}
+```
+
+Next, in your standard view configuration file include a block that indicates when this Controller will be used. It is similar to regular view config, but contains additional information:
```

Comment:
'standard view configuration file' not sure what it is, could you add a path to it.
Is this 'block' a stanalone entity or will it be placed inside already existing code. If it is the second scennario we could use information under what key and inline with what key.

Reply:
'standard view config file' can be a lot of things (you can place the config in ezplatform.yml, or in a separate file), so it depends on each setup and there's no place here to go into details.